e2e/src/usecases/PINChange.js

```diff
@@ -24,6 +24,15 @@ export default ChangePIN = () => {
     // Then we enter the new PIN
     await enterPinUi(ALTERNATIVE_PIN)
     await sleep(500)
+
+    // Enter an invalid pin and check that we get the correct error and start over
+    await enterPinUi('902100')
+    await expect(element(by.text("The PINs didn't match"))).toBeVisible()
```

Comment:
What in Linter's name?!  (the double quotes)

Reply:
Can change to `'The PINs didn\'t match'`.

Reply:
ohhh, i missed that.  sorry.  looks fine.